telegram/message.py

```diff
@@ -2970,6 +2991,9 @@ def caption_markdown_v2_urled(self) -> str:
         Returns:
             :obj:`str`: Message caption with caption entities formatted as Markdown.
 
+        .. versionchanged:: 13.10
+           Spoiler entities are now formatted as Markdown V2.
+
```

Comment:
maybe also a `.. versionchanged::` for `text/caption_markdown` along the lines "raises ValueError if the message contains spoiler entities"? and a `Raises:` block, if it doesn't have that yet …